CFC_DataCollector/recommender/utility_model_pipeline.py

```diff
@@ -0,0 +1,17 @@
+"""
+Construct user utility model or retrieve from database and update with
+augmented trips. Store in database and return the model.
+"""
+
+from user_utility_model import UserUtilityModel
+
+def build_user_model(user_id, augmented_trips):
+  model = UserUtilityModel.find_from_db(user_id)
+
+  if model:
+    model.update(augmented_trips)
+  else:
+    model = UserUtilityModel(user_id, augmented_trips)
```

Comment:
While making pipeline 3, Lesley, Gautham and I decided that the generation of modelPrime (modelPrime = model + weight for carbon) should be part of the model pipeline. The rationale was that we might end up with fairly complex models (a learnt model and a specified model, multiple learnt models based on different features, etc) and it didn't make sense to glom all that into the third pipeline, which was basically about calculating the utility of the recommendations given a model.

So basically the model pipeline would do whatever the heck it wanted to do and save the one model that the third pipeline should use into the database (it can also store multiple other models in other databases). The third pipeline will just read that model and apply it to get the various utilities.

Makes sense?